paddle/fluid/framework/details/fetch_op_handle.cc

```diff
@@ -61,9 +61,14 @@ void FetchOpHandle::RunImpl() {
     auto &scope = scopes[i];
     auto *var =
         scope->FindVar(kLocalExecScopeName)->Get<Scope *>()->FindVar(var_name);
+    if (var == nullptr) {
+      scope->FindVar(var_name);
```

Comment:
This line has no effect

Reply:
Oh, right, `scope->FindVar(kLocalExecScopeName)->Get<Scope *>()` is the subscope of `scope`.